Makefile

```diff
@@ -95,7 +95,17 @@ ifeq ($(BUILDOS),linux)
 endif
 
 
-.PHONY: livepeer livepeer_bench livepeer_cli livepeer_router docker swagger
+.PHONY: ai_worker_codegen livepeer livepeer_bench livepeer_cli livepeer_router docker swagger
+
+# Git reference to download the OpenAPI spec from, defaults to `main` branch.
+# It can also be a simple git commit hash. e.g. `make ai_worker_codegen REF=c19289d`
+REF ?= refs/heads/main
+ai_worker_codegen:
```

Comment:
@victorges I assume this command is excluded from `all` to prevent unexpected breakage? If so, perhaps we should document it in the developer docs?